modules/subscriptions/subscriptions/subscriptions-settings.tsx

```diff
@@ -63,31 +99,22 @@ const PaymentHistory: React.FC = () => {
     },
   ];
 
-  const [data, setData] = useState<GenesisInvoice[] | []>([]);
-  const [currentPage, setCurrentPage] = useState(1);
-  const [isLoading, setIsLoading] = useState(false);
-  const [totalRecords, setTotalRecords] = useState(0);
-
-  const itemsPerPage = 5;
-  const fromRange = currentPage * itemsPerPage - itemsPerPage;
-  const toRange = fromRange + itemsPerPage - 1;
-  const totalPages = Math.ceil(totalRecords / itemsPerPage);
-
-  const loadData = () => {
-    void (async () => {
```

Comment:
We remove all this things, pagination now is broken, I can not load page 2.

Why we require to change so much code? When is not related with the bug. 
is scary and force us to re-test things that are stable